mapstruct-2/src/test/java/com/baeldung/mapstruct/date/mapper/DateMapperTest.java

```diff
@@ -0,0 +1,35 @@
+package com.baeldung.mapstruct.date.mapper;
+
+import org.junit.jupiter.api.Test;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class DateMapperTest {
```

Comment:
This test is not using mapstruct at all

Reply:
This is for testing the general conversion method which is designed to be utilized by ReuseUserMapper.

Reply:
I'm not sure this is a valuable test for this article scope, I'd prefer we keep the tests from the perspective of a developer using mapstruct